app/src/main/kotlin/no/nav/syfo/infrastructure/infotrygd/InfotrygdService.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.syfo.infrastructure.infotrygd
+
+import no.aetat.arena.arenainfotrygdskjema.ObjectFactory
+import no.nav.syfo.application.mq.MQSender
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.infrastructure.mq.JAXB
+import java.math.BigInteger
+import java.time.LocalDate
+import javax.xml.datatype.DatatypeFactory
+
+class InfotrygdService(
+    val mqQueueName: String,
+    val mqSender: MQSender,
+) {
+
+    fun sendMessageToInfotrygd(
+        personident: PersonIdent,
+        veilederId: String,
+        navKontor: String,
```

Comment:
`veilederIdent`*
Og per i dag har vi ikke snakket om å lagre ned nav-kontor, men hvis vi trenger dette for å sende til infotrygd så burde vi kanskje sende med dette fra frontenden også?

Reply:
Ja, for nav-kontor/enhet kan vi ikke hente ut av token'et antakelig.

Vi må snakke litt mer med infotrygd om dette: jeg føler meg ikke helt trygg på at de kan bruke azure-veilederIdent'ene.

Reply:
Hmm, [her](https://nav-it.slack.com/archives/C9LBDCYBH/p1709035215668129?thread_ts=1707917046.912759&cid=C9LBDCYBH) er et eksempel på XML som de sender, men ser ingen VeilederId der?

Reply:
Ja, det var det feltet jeg tenkte skulle være veilederident, men det er kanskje noe annet ja. Vi får tilpasse etter hvert.